src/runtime_src/core/tools/xbutil2/CMakeLists.txt

```diff
@@ -100,6 +100,8 @@ target_link_libraries(${XBUTIL2_NAME}
 
 if (NOT WIN32)
   target_link_libraries(${XBUTIL2_NAME} PRIVATE pthread uuid dl)
+else()
+  target_link_libraries(${XBUTIL2_NAME} PRIVATE ws2_32)
```

Comment:
Why is  ws2_32 neeeded?

Reply:
ws2_32 is included here and used by some of the binaries: https://github.com/Xilinx/XRT/blob/c34c5afb4c92cb68460349185da7e8fb24f9d957/src/runtime_src/core/tools/common/XBUtilities.cpp#L32